code/modules/reagents/chemistry/holder.dm

```diff
@@ -264,10 +264,10 @@
 	return TRUE
 
 /// Like add_reagent but you can enter a list. Format it like this: list(/datum/reagent/toxin = 10, "beer" = 15)
-/datum/reagents/proc/add_reagent_list(list/list_reagents, list/data=null)
+/datum/reagents/proc/add_reagent_list(list/list_reagents, list/data=null, reagtemp)
```

Comment:
```suggestion
/datum/reagents/proc/add_reagent_list(list/list_reagents, list/data, reagtemp = DEFAULT_REAGENT_TEMPERATURE)
```

Otherwise stuff such as food will spawn with reagents at absolute zero temperature.

Reply:
![image](https://user-images.githubusercontent.com/51932756/157111466-14649023-b730-48f2-9ba4-7415e337fe8d.png)
They haven't been doing this but sure